plugin/cypress/support/commands.ts

```diff
@@ -25,13 +25,29 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 //
-// declare global {
-//   namespace Cypress {
-//     interface Chainable {
-//       login(email: string, password: string): Chainable<void>
-//       drag(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
-//       dismiss(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
-//       visit(originalFn: CommandOriginalFn, url: string, options: Partial<VisitOptions>): Chainable<Element>
-//     }
-//   }
-// }
\ No newline at end of file
+declare global {
+    namespace Cypress {
+        interface Chainable {
+            login(OC_CLUSTER_USER: string, OC_CLUSTER_PASS: string, OC_IDP: string): Chainable<void>
+            //   drag(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
+            //   dismiss(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
+            //   visit(originalFn: CommandOriginalFn, url: string, options: Partial<VisitOptions>): Chainable<Element>
```

Comment:
This just looks like boilerplate that can be removed.

Reply:
> This just looks like boilerplate that can be removed.

Will remove with the next commit.